js/src/components/Welcome/Welcome.tsx

```diff
@@ -1,5 +1,7 @@
 import { Title, Text, Anchor } from '@mantine/core'
 import classes from './Welcome.module.css'
+import { RootRoutes } from '../../Routes'
```

Comment:
We should really get aliasing working when we can.

Reply:
will do in a separate PR